sdk/dapp-kit/src/hooks/useSuiClientInfiniteQuery.ts

```diff
@@ -24,7 +24,11 @@ export type SuiRpcPaginatedMethods = {
 	}>
 		? {
 				name: K;
-				result: R;
+				result: {
+					data?: R;
```

Comment:
Do we want to unpack this and just have `hasNextPage` and `nextCursor` for internal usage instead?